frontend/packages/git-service/src/services/__tests__/bitbucket-service.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { GitSource, RepoFileList } from '../../types';
+import { BitbucketService } from '../bitbucket-service';
+import { DockerFileParser } from '../../utils';
+
+describe('Bitbucket Service', () => {
+  // For some reason this test passes locally but fails on the CI
+  xit('should list all files of existing public bitbucket repo', (done: any) => {
```

Comment:
nit: will mock help here rather than making an actual call

Reply:
Not sure if mocking will solve the problem of tests not passing on CI. But i kept the tests to make actual calls since the library depends on actual REST APIs of Github, Bitbucket and Gitlab. Tests would help us if something changes in their API. @christianvogt What do you think?

Reply:
I would prefer that all tests that reach out through the network get mocked. We can't be sure that the CI will always provide network access.

Reply:
I'll mock the async API calls for tests then.